analyzer/src/main/kotlin/managers/DotNet.kt

```diff
@@ -31,12 +31,17 @@ import org.ossreviewtoolkit.analyzer.AbstractPackageManagerFactory
 import org.ossreviewtoolkit.analyzer.PackageManager
 import org.ossreviewtoolkit.analyzer.managers.utils.NuGetDependency
 import org.ossreviewtoolkit.analyzer.managers.utils.NuGetSupport
-import org.ossreviewtoolkit.analyzer.managers.utils.OPTION_DIRECT_DEPENDENCIES_ONLY
+import org.ossreviewtoolkit.analyzer.managers.utils.NugetInspector
```

Comment:
IMHO NuGet.kt is obsolete and should be removed entirely.
https://github.com/oss-review-toolkit/ort/blob/14bf7f5079b8d751c2a3cf4adf2d311b25116337/analyzer/src/main/kotlin/managers/NuGet.kt supports parsing `packages.config`. This is already supported by `nuget-inspector`, but slightly differently: `packages.config` never exist alone in the void, they exist only in the context of another project file. You can point nuget-inspector to a packages.config file and it will search for the companion project file.

Reply:
Then I'd argue we should to the opposite: Keep `NuGet.kt` and remove `DotNet.kt`, as that would also more nicely fit the naming of `nuget-inspector`.

Reply:
@TG1999 what about this?
Let's delete NuGet.kt and move "packages.config" to the list of supported files in "DotNet.kt", since this is a dotnet file anyway.

Reply:
Done, we kept NuGet.kt afterall.